Streetcode/Streetcode.BLL/MediatR/AdditionalContent/Subtitle/GetByStreetcodeId/GetSubtitleByStreetcodeIdQueryHandler.cs

```diff
@@ -0,0 +1,32 @@
+﻿using AutoMapper;
+using FluentResults;
+using MediatR;
+using Streetcode.BLL.DTO.AdditionalContent.Subtitles;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+
+namespace Streetcode.BLL.MediatR.Subtitle.GetByStreetcodeId
+{
+    public class GetSubtitleByStreetcodeIdQueryHandler : IRequestHandler<GetSubtitleByStreetcodeIdQuery, Result<SubtitleDTO>>
+    {
+        private readonly IMapper _mapper;
+        private readonly IRepositoryWrapper _repositoryWrapper;
+
+        public GetSubtitleByStreetcodeIdQueryHandler(IRepositoryWrapper repositoryWrapper, IMapper mapper)
+        {
+            _repositoryWrapper = repositoryWrapper;
+            _mapper = mapper;
+        }
+
+        public async Task<Result<SubtitleDTO>> Handle(GetSubtitleByStreetcodeIdQuery request, CancellationToken cancellationToken)
+        {
+            var subtitle = await _repositoryWrapper.SubtitleRepository.GetFirstOrDefaultAsync(Subtitle => Subtitle.StreetcodeId == request.streetcodeId);
```

Comment:
Please space out the statement (lucky you StyleCop setup doesn't have max-line constraint).